lightkurve/interact.py

```diff
@@ -397,9 +403,30 @@ def show_interact_widget(tpf, notebook_url='localhost:8888',
         will need to supply this value for the application to display
         properly. If no protocol is supplied in the URL, e.g. if it is
         of the form "localhost:8888", then "http" will be used.
-    max_cadences : int
+    max_cadences: int
         Raise a RuntimeError if the number of cadences shown is larger than
         this value. This limit helps keep browsers from becoming unresponsive.
+    transform_func: function
+        A function that transforms the lightcurve.  The function takes in a
+        LightCurve object as input and returns a LightCurve object as output.
+        For example, the function could be to normalize the lightcurve:
+
+            transform_func = lambda lc: lc.normalize()
+
+        Or it could be more complex, such as detrending the lightcurve.  In this
+        way, the interactive selection of aperture mask can be evaluated after
+        inspection of the transformed lightcurve.  See the tutorial for some
+        limitations. The transform_func is applied before saving a fits file.
```

Comment:
I'd remove the sentence `See the tutorial for some limitations.` for now, unless you can link to a specific tutorial?